tests/ci/slow_tests_8x.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip install --upgrade pip
+export RUN_SLOW=true
```

Comment:
where this RUN_SLOW is used?

Reply:
In `tests/test_examples.py`, at line 120, there is a decorator called `slow` that wraps all non-regression tests. With this decorator, you need to declare the environment variable `RUN_SLOW` as true so that these tests are not skipped.
It basically avoids running all those long tests if one launches `pytest tests/`.